src/apps/schema/src/app/components/AddFieldModal/TooltipBody.tsx

```diff
@@ -66,7 +68,12 @@ export const TooltipBody = ({
           {description}
         </Typography>
         <Box py={2}>
-          <Typography variant="body2" fontWeight="700">
+          <Typography
+            lineHeight={"20px"}
```

Comment:
It's indicated on figma that this heading should have that 20px line-height instead of the default one. 

Do you think i should not add it here since it's not part of the request? I just notice its off by few pixels

Reply:
You're right @zcolah  Thank you! 

@agalin920  reverthing this changes